index.html

```diff
@@ -65,6 +65,217 @@ <h3><a> Let's connect </a> </h3>
         </div>
     </section>
 
+    <section class="portfolio-1">
+        <div class="project-1">
+            <div class="tonic-img-1">
+                <img src="Image/SnapshootPortfolio.png" alt="error">
+            </div>
+            <div class="project-name-1">
+                <h1>Tonic</h1>
+            </div>
+            <div class="about-project-1">
+                <ul>
+                    <li class="company">CANOPY</li>
+                    <li class="position">Back End Dev</li>
+                    <li class="year">2015</li>
+                </ul>
+            </div>
+            <div class="role-project-1">
+                <p>
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="skills-project-1">
+                <ul>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <br>
+            <br>
+            <div class="view-project-1">
+                <button type="button"> See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <section class="portfolio-2">
+        <div class="project-2">
+            <div class="tonic-img-2">
+                <img src="Image/SnapshootPortfolio2.png" alt="error">
+            </div>
+            <div class="project-name-2">
+                <h1>Multi-Post Stories</h1>
+            </div>
+            <div class="about-project-2">
+                <ul>
+                    <li class="company-2">CANOPY</li>
+                    <li class="position-2">Back End Dev</li>
+                    <li class="year-2">2015</li>
+                </ul>
+            </div>
+            <div class="role-project-2">
+                <p>
+                    A daily selection of privately personalized reads;
+                    no accounts or sign-ups required.
+                </p>
+            </div>
+            <div class="skills-project-2">
+                <ul>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>javascript</li>
+                </ul>
+            </div>
+            <br>
+            <br>
```

Comment:
- [ ] Kindly delete all `<br>` tags, it is not semantic, use margin, padding to align content 👍🏿